account-management-api/src/main/java/uk/gov/di/accountmanagement/lambda/AuthoriseAccessTokenHandler.java

```diff
@@ -49,17 +53,26 @@ public AuthPolicy handleRequest(TokenAuthorizerContext input, Context context) {
         LOGGER.info("Request received in AuthoriseAccessTokenHandler");
         try {
             String token = input.getAuthorizationToken();
+            AccessToken accessToken = AccessToken.parse(token, AccessTokenType.BEARER);
+            JWTClaimsSet claimsSet = SignedJWT.parse(accessToken.getValue()).getJWTClaimsSet();
 
-            AccessToken accessToken;
-            accessToken = AccessToken.parse(token, AccessTokenType.BEARER);
-            String subject = SignedJWT.parse(accessToken.getValue()).getJWTClaimsSet().getSubject();
             boolean isAccessTokenSignatureValid =
                     tokenValidationService.validateAccessTokenSignature(accessToken);
             if (!isAccessTokenSignatureValid) {
                 LOGGER.error("Access Token signature is not valid");
                 throw new RuntimeException("Unauthorized");
             }
             LOGGER.info("Successfully validated Access Token signature");
+
+            List<String> scopeList = claimsSet.getStringListClaim("scope");
```

Comment:
In theory it should never be null but is it worth handling any null pointer is scope or subject is not present?

Reply:
claimsSet.getStringListClaim() returns null for a claim that isn't there, so the null check on line 68 handles this.  Have added a null check and error for the subject.